decorator.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
```

Comment:
- [ ] Please as the last reviewer explained make the namable attribute accessible by  instance by adding it to attr_accessor